packages/components/src/Switch/Switch.mdx

```diff
@@ -51,6 +61,8 @@ import { Switch } from "@jobber/components/Switch";
 
 ## Disabled
 
+A disabled switch cannot be operated by the user. If presenting a disabled "On" switch to the user, the context and ability for the user to enable the switch must be apparent to avoid creating a sense that the user has lost control of the interface.
```

Comment:
I would just adjust the copy to be a bit less clinical sounding: instead of "the context and ability for the user to enable the switch must be apparent" can we just simply say "Provide clear description for the user on how to enable the switch."